CATALOG-REFERENCE.md

```
@@ -191,10 +193,11 @@ both paths above). All base tables are queryable (working).
 `Reflects` = does it show runtime-registered user objects (`live`/`merge`/`static`,
 see above). `Reads from` = the main catalog tables the view's SQL joins.
 
-### Working (18)
+### Working (19)
 | View | Purpose | Reads from | Reflects |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)